dev/io.openliberty.org.jboss.weld5/bnd.overrides

```diff
@@ -7,23 +7,194 @@ bFullVersion=${version;==;${bVersion}}.${libertyBundleMicroVersion}
+ org.jboss.weld.exceptions;version="5.0.0.Beta1";uses:="jakarta.enterprise.inject,jakarta.enterprise.inject.spi",\
+ org.jboss.weld.injection;version="5.0.0.Beta1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,jakarta.enterprise.inject.spi.configurator,org.jboss.weld.bootstrap.api,org.jboss.weld.injection.spi,org.jboss.weld.manager",\
+ org.jboss.weld.interceptor.proxy;version="5.0.0.Beta1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,jakarta.interceptor,org.jboss.weld.bean.proxy,org.jboss.weld.interceptor,org.jboss.weld.interceptor.spi.metadata,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager,org.jboss.weld.util",\
+ org.jboss.weld.interceptor.util.proxy;version="5.0.0.Beta1";uses:="org.jboss.weld.bean.proxy",\
+ org.jboss.weld.logging;version="5.0.0.Beta1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,javax.naming,org.jboss.logging,org.jboss.weld.contexts,org.jboss.weld.exceptions",\
+ org.jboss.weld.manager;version="5.0.0.Beta1";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.inject.spi.configurator,org.jboss.weld.bean,org.jboss.weld.bean.proxy,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi,org.jboss.weld.construction.api,org.jboss.weld.contexts,org.jboss.weld.ejb.spi,org.jboss.weld.event,org.jboss.weld.events,org.jboss.weld.inject,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager.api,org.jboss.weld.serialization.spi",\
+ org.jboss.weld.module;version="5.0.0.Beta1";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,org.jboss.weld.bean,org.jboss.weld.bean.builtin,org.jboss.weld.bootstrap,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi,org.jboss.weld.ejb.spi,org.jboss.weld.event,org.jboss.weld.manager",\
+ org.jboss.weld.security;version="5.0.0.Beta1";uses:="org.jboss.weld.security.spi",\
+ org.jboss.weld.util;version="5.0.0.Beta1";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.util,jakarta.interceptor,org.jboss.weld.bean,org.jboss.weld.bootstrap,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi,org.jboss.weld.exceptions,org.jboss.weld.inject,org.jboss.weld.injection,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager,org.jboss.weld.resources.spi,org.jboss.weld.serialization.spi",\
+ org.jboss.weld.xml;version="5.0.0.Beta1";uses:="org.jboss.weld.bootstrap.spi,org.xml.sax,org.xml.sax.helpers"
 
+# The first 2 imported packages are additions, the rest come from the original manifest
 Import-Package: \
  javax.security.auth,\
  jakarta.xml.ws;version="3.0",\
```

Comment:
Should this be a range instead of just 3.0

Reply:
Hmm now you mention it, I don't know why that import is there. It has clearly just been copied from the previous version. I understand why an import for `javax.security.auth` is needed but not `jakarta.xml.ws`???